### DISCUSSION TITLE
When will I get the benefits of Github Education?

### DISCUSSION THREAD

**Discussion by barankut** [community/community#137614] [GitHub Education] [answered]
### Hi everyone,

I already got approved a couple days ago but still didn't get the benefits. Do I have to do something?

**Comment by davevad93** [comment, marked as answer]
Hi @barankut ,

Don't worry, you don't have to do anything, just wait; when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @barankut was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by barankut** [reply]
Thank you for the relief. Have a good day.

**Reply by davevad93** [reply]
You're welcome @barankut have a nice day too! 👍

**Reply by davevad93** [reply]
Hey @Ahmed-Swilam, when your the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Ahmed-Swilam was approved and your benefits are now available.`

Waiting time may vary and there's nothing that can make it shorter! 👍
